tests/Processor/SanitizeDataProcessorTest.php

```diff
@@ -157,7 +159,9 @@ public function testOverridenSanitize($processorOptions, $client_options, $dsn)
         /**
          * @var SanitizeDataProcessor $processor
          */
-        $processor = $client->processors[0];
+        $processors = new \ReflectionProperty($client, 'processors');
```

Comment:
PHPUnit provides an `assertAttributeEquals` method to check whether an attribute (even private) equals a given value